app/javascript/components/editor-config/citations.js

```diff
@@ -89,20 +91,23 @@ export const addPostLink = function (state, dispatch, view) {
 
     const handleClose = () => ReactDOM.unmountComponentAtNode(root)
 
-    const handleNewPostLink = (payload) => {
-      console.log('in handleNewPostLink this is the payload', payload)
+    // TODO: refactor
```

Comment:
what does this comment mean?

Reply:
There is repetitive code in `handleNewCitation` that can be made concise. Right now I am not sure how to refactor while preserving the Promise.

I removed the TODO.